docs/book/component-gallery/experiment-trackers/neptune.md

```diff
@@ -0,0 +1,183 @@
+track and visualize the results produced by your
+automated pipeline runs, as you make the transition towards a more production
+oriented workflow.
+
+You should use the Neptune Experiment Tracker:
+* if you have already been using neptune.ai to track experiment results
+for your project and would like to continue doing so as you are incorporating 
+MLOps workflows and best practices in your project through ZenML.
+* if you are looking for a more visually interactive way of navigating the
+results produced from your ZenML pipeline runs (e.g. models, metrics, datasets)
+* if you would like to connect ZenML to neptune.ai to share the artifacts
+and metrics logged by your pipelines with your team, organization or external
+stakeholders
+
+You should consider one of the other [Experiment Tracker flavors](./experiment-trackers.md#experiment-tracker-flavors)
+if you have never worked with neptune.ai before and would rather use
+another experiment tracking tool that you are more familiar with.
+
+## How do you deploy it?
+
+The Neptune Experiment Tracker flavor is provided by the Neptune-ZenML
+integration. You need to install it on your local machine to be able to register
+the Neptune Experiment Tracker and add it to your stack:
+
+```shell
+zenml integration install neptune -y
+```
+
+The Neptune. Experiment Tracker needs to be configured with the
+credentials required to connect to Neptune using an API token.
+
+### Authentication Methods
+
+You need to configure the following credentials for authentication to
+Neptune:
+
+* `api_token`: API key token of your Neptune account. If left blank, Neptune will
+attempt to retrieve it from your environment variables.
+* `project`: The name of the project where you're sending the new run, in the form "workspace-name/project-name".
+If the project is not specified, Neptune will attempt to retrieve it from your environment variables.
+
+
+{% tabs %}
+{% tab title="Basic Authentication" %}
+
+This option configures the credentials for neptune.ai
+directly as stack component attributes.
+
+{% hint style="warning" %}
+This is not recommended for production settings as the credentials won't be
+stored securely and will be clearly visible in the stack configuration.
+{% endhint %}
+
+```shell
+# Register the Neptune experiment tracker
+zenml experiment-tracker register neptune_experiment_tracker --flavor=neptune \ 
+    --project=<project_name> --api_token=<token>
+
+# Register and set a stack with the new experiment tracker
+zenml stack register custom_stack -e neptune_experiment_tracker ... --set
+```
+{% endtab %}
+
+{% tab title="Secrets Manager (Recommended)" %}
+
+This method requires you to include a [Secrets Manager](../secrets-managers/secrets-managers.md)
+in your stack and configure a ZenML secret to store the neptune.ai
+credentials securely.
+
+{% hint style="warning" %}
+**This method is not yet supported!**
+
+We are actively working on adding Secrets Manager support to the Neptune
```

Comment:
Correct, it's even specified correctly as `api_token: Optional[str] = SecretField()` in the config. I just checked and this is probably copied from the MLflow experiment tracker docs, which say the same thing (not yet supported). We should probably in both cases just link to the secrets page?

Reply:
@htahir1 I see this as something that one of us should write instead of @AleksanderWWW, as it's not really related to the integration and requires knowledge about secrets/secret managers. wdyt?

Reply:
Ok. I think that the remaining suggestions have been implemented